src/requirements/data/colleges/en.ts

```diff
@@ -38,13 +38,14 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
   {
     name: 'Physics',
     description:
-      'PHYS 1112 and 2213, and, depending on the Major, either PHYS 2214 or a designated mathematics or science course.',
+      'The normal program in physics includes PHYS 1112, PHYS 2213, and PHYS 2214 or the corresponding honors courses (PHYS 1116, PHYS 2217, and PHYS 2218).' +
```

Comment:
Can you get rid of the start of this description? Specifically - "The normal program in physics includes ". I think it just adds extra words when we can just say the course info.

Reply:
Also - we can keep the second half of the wording the same?

Reply:
ok!